src/ocean/util/app/DaemonApp.d

```diff
@@ -460,7 +463,8 @@ public abstract class DaemonApp : Application,
         this.registerExtension(this.timer_ext);
 
         // Register stats timer with epoll
-        this.timer_ext.register(&this.statsTimer, this.stats_ext.config.interval);
+        ulong offset = timeToNextInterval(this.stats_ext.config.interval);
```

Comment:
`this.` :angel:

Reply:
It's `static` :P

Reply:
In the got this email, that suddenly appeared to me :-) :+1: